common/djangoapps/student/management/tests/test_bulk_user_activate.py

```diff
@@ -0,0 +1,67 @@
+"""Tests for the bulk_user_activate command."""
+
+import ddt
+from django.core.management import call_command
+from django.core.management.base import CommandError
+
+from student.tests.factories import UserFactory
+from student.models import User
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
```

Comment:
Don't need to use `SharedModuleStoreTestCase` which is for tests that need a course, just use the base Django testcase: `from django.test import TestCase`